packages/garbo/src/fights.ts

```diff
@@ -1297,6 +1299,8 @@ const freeRunFightSources = [
           .trySkillRepeat(
             $skill`Fire Extinguisher: Polar Vortex`,
             $skill`Perpetrate Mild Evil`,
+            // eslint-disable-next-line libram/verify-constants
+            $skill`Swoop like a Bat`,
```

Comment:
are we nearing the round limit with this?

Reply:
I don't know. We could be. The plan is to test it tomorrow and find out.

Reply:
To answer your question, it didn't swoop! So I still don't know. I think Libram/eslint need to be updated first to test that.

Reply:
No, you just never wrote code to equip the wings.